lib/3scale_toolbox/commands/service_command/service_update.rb

```diff
@@ -0,0 +1,28 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceUpdateSubcommand < Cri::CommandRunner
```

Comment:
Does it make sense to have a service update?
I don't have an idea on what would be useful for. For the moment I've left the skeleton of the subcommand here

Reply:
For the moment we are not going to implement this command unless we receive feedback that is needed with some use cases being used by users